src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -15,3 +15,15 @@ databaseChangeLog:
   - include:
       file: changesets/changelog_2022-07-06T09:38:41Z.xml
       relativeToChangelogFile: true
+
+  - include:
+      file: changesets/changelog_2022-10-13T15:32:28Z.xml
+      relativeToChangelogFile: true
+
+  - include:
+      file: changesets/changelog_2022-10-14T11:37:50Z.xml
+      relativeToChangelogFile: true
+
+  - include:
+      file: changesets/changelog_2022-10-14T16:22:13Z.xml
+      relativeToChangelogFile: true
```

Comment:
you could probably merge the changelogs into 1

Reply:
yes, I will try to generate a single log in next commit.